test/integration/patients/patient/archive.js

```diff
@@ -56,6 +57,7 @@ context('patient archive page', function() {
               state: getRelationship(stateDone),
               form: getRelationship(testForm),
               files: getRelationship([{ id: '1' }], 'files'),
+              comments: getRelationship(getComment()),
```

Comment:
seems like this should either be `comments: getRelationship([])` or `comments: getRelationship([getComment()])`

Reply:
Yep, that's invalid. Fixed those. Not sure why that didn't cause the tests to fail though...

Reply:
And should we add the `'comments'` type?

Like `comments: getRelationship([getComment()], 'comments')`.